Classes/Domain/Dto/Exif.php

```diff
@@ -11,88 +11,259 @@
  * source code.
  */
 
-/*
- * Exif DataType
- * See: http://www.exif.org/Exif2-2.PDF
+/**
+ * EXIF DataType
+ * @see http://www.cipa.jp/std/documents/e/DC-008-Translation-2016-E.pdf
  */
 class Exif extends AbstractMetaDataDto
 {
+    /**
+     * EXIF MetaData constructor
+     *
+     * @param array $properties
+     */
+    public function __construct(array $properties)
+    {
+        $this->properties = [
+            'ImageWidth' => 0,
+            'ImageLength' => 0,
+            'BitsPerSample' => [0, 0, 0],
+            'Compression' => '',
+            'PhotometricInterpretation' => '',
+            'Orientation' => '',
+            'SamplesPerPixel' => 0,
+            'PlanarConfiguration' => '',
+            'YCbCrSubSampling' => '',
+            'YCbCrPositioning' => '',
+            'XResolution' => 0.0,
+            'YResolution' => 0.0,
+            'ResolutionUnit' => '',
+            // 'StripOffsets' => ?,
+            // 'RowsPerStrip' => ?,
```

Comment:
yeah. I don't think they will be used and it wasn't immediately clear to me how to represent these. Do you have an idea? If not they can be removed entirely.